### PR TITLE
Chirret species

### PR DESCRIPTION
**Pull request by Cameron653**
Adds Chirrets (Sprited by Natje and others. Coded by me.)
- Immune to cold temperatures
- Become confused in hot temperatures, pass out, and eventually take internal organ damage over time.
- Slight pressure resistance
- A large amount of special organs that have custom sprites!
- Able to make web cocoons (resprited bodybags) https://i.imgur.com/AL0yfrA.png
- Able to see in the dark
- Able to move as fast as if they had Haste (non-plus variant)
- 80% total health as opposed to 100%
- 25% more damage from brute
- 25% less damage from burns
- Sugar will quickly cause chirrets to become intoxicated (drunk)
- Coffee causes chirrets to become extremely jittery
- Given some neutral traits (HV since claws, trashcan since they're not picky eaters.)
- Has teshari cough and sneeze since it's cute
- Has access to a *chirp command
- Multiple markings for complete customization! Two variants for stomach/head markings. Cheek markings! Dual colored tail! Claw markings!

Clothed icons
https://i.imgur.com/kyyK3Hq.png North
https://i.imgur.com/KWW3vvy.png East
https://i.imgur.com/hYgxQFM.png South
https://i.imgur.com/ZLSgEyK.png West

Naked icons
https://i.imgur.com/MRLCwAP.png (North South West)
https://i.imgur.com/fHtqBgm.png (East.) (With cheek marking and no belly marking)

### CODE REVIEW

**Comment by SpadesNeil** [comment]
Conflicts aaaa but there's gonna be more conflicts before this could be merged so hold off until https://github.com/VOREStation/VOREStation/pull/2558 is resolved.

**Comment by Cameron653** [comment]
Alright. I can fix the conflicts by downloading the .dmi file, slapping it in, and then copying all my changes over to the master branch .dmi file whenever that PR is merged.

File: code/modules/mob/living/carbon/human/emote_vr.dm [hunk 1]
```diff
@@ -51,6 +51,14 @@
 				SC.emote_into(nifmessage,src)
 			return 1
 
+		if("chirp")
+			if(src.species.name == "Chirret")
+				message = "<B>The [src.name]</B> chirps!"
+				playsound(src.loc, 'sound/misc/nymphchirp.ogg', 50, 0)
+				m_type = 2
+			else
+				src << "<span class='warning'>You can't chirp without being a Chirret!</span>"
```
Comment (Belsima) [inline code/modules/mob/living/carbon/human/emote_vr.dm]: to_chat

Comment (Cameron653) [reply code/modules/mob/living/carbon/human/emote_vr.dm]: Resolved.

**Comment by Verkister** [comment]
The currently open conflicting PR will probably have to wait til this one's in because it has way outdated files in it (from all the way back to october) and needs updates anyway. The merged conflict is in the markings file that fixed cow body markings and added faux sergal eyes and vulp stripe.

**Comment by Cameron653** [comment]
...Hmm. Command prompt didnt' fix it. One second.

**Comment by Cameron653** [comment]
There we go! Solved it. 

Took the markings_vr file in the master, pasted the stuff into it and put it in this PR.

**Comment by SpadesNeil** [comment]
I guess https://github.com/VOREStation/VOREStation/pull/2558 will have to wait.

**Comment by SpadesNeil** [comment]
What exactly are we doing about clothes though and is there a lore page for them yet?

**Comment by Cameron653** [comment]
Clothing (for the most part) work for them, although some of it is sticking out.

I'm not sure about lore.

**Comment by SpadesNeil** [comment]
This is being reverted.

**Pull request merged**